CHANGE-NOTES.html

```diff
@@ -1,4 +1,7 @@
-<p>Fix docs on the shortcut that opens Git Machete Tab <i>(command option shift M)</i> on Mac.</p>
+<p>Dropped support for IntelliJ 2020.2.</p>
+
+
+<p>The <i>Pull</i> action on specific branch displays notification about skipping fetch action when the last fetch was performed less than a minute ago.</p>
```

Comment:
The changes to code refer fast forward merge - not the pull. 🤔 Probably it's applicable too but for now inconsistent.

Reply:
Let's "stash away" this PR for a moment and revisit after/apply on the top of changes needed for issue #852